<launch>
  
  <include file="$(find simulation_environment)/launch/lawn_mower_full.launch">
	<arg name="numSensors" value="2"/>
  </include>
  
  <node pkg="control" name="wallFollower" type="wallFollower">
	  <rosparam command="load" file="$(find interfaces)/config/config.yaml"/>
  </node>
  
  <node pkg="mapping" name="dpGeneration" type="dpGeneration" output="screen">
	  <rosparam command="load" file="$(find interfaces)/config/config.yaml"/>
  </node>
  
  <node pkg="mapping" name="lcDetection" type="lcDetection" output="screen">
	  <rosparam command="load" file="$(find interfaces)/config/config.yaml"/>
  </node>
  
  <node pkg="mapping" name="pgOptimization" type="pgOptimization" output="screen">
	  <rosparam command="load" file="$(find interfaces)/config/config.yaml"/>
  </node>
  
  <!-- Start rosbag to store data 
  <node pkg="rosbag" name="test_bag" type="record" 
		args="-o ../bag_files/robotData /controlData /odometryData /imuData /sensorData"/>
		-->

</launch>
